data/bedrock/latest/proto.yml

```diff
@@ -652,7 +649,7 @@ packet_move_player:
          source_entity_type: LegacyEntityType
    tick: varint64
 
-packet_rider_jump:
+packet_passenger_jump:
```

Comment:
Please don't rename packets unless it's a different packet (same for fields holding same data), this unnecessarily breaks code compatibility

Reply:
But if there does need to be a rename for something, it should be backported to the old versions. I've fixed this one.